README/RCodingStyle.hxx

```diff
@@ -174,5 +174,5 @@ public:
 double Add(const TExampleClass &a, const TExampleClass &b);
 
 } // namespace ROOT
-#endif // #ifndef ROOT_RCodingStyle
+#endif // ROOT_RCodingStyle
```

Comment:
Commenting on the commit message... Maybe a `Suggested-by: ` tag in the body of the commit would be better?